scripts/installer

```diff
@@ -43,10 +43,10 @@ initOS() {
 }
 
 verifySupported() {
-  if ! type "kubectl" > /dev/null; then
-    echo "kubectl is required"
-    exit 1
-  fi
+  # if ! type "kubectl" > /dev/null; then
+  #   echo "kubectl is required"
+  #   exit 1
+  # fi
```

Comment:
```suggestion
  if ( [ "$ACTION" == "install" ] || [ "$ACTION" == "uninstall" ] ) && ! type "kubectl" &> /dev/null; then
    echo "kubectl is required"
    exit 1
  fi
```

Reply:
Updated to use `2>&1` instead of the `&>` shorthand since it's POSIX compliant and already used elsewhere